### ISSUE TITLE
Report failling with vitest 0.25.5

### ISSUE THREAD

**Issue by DamnClin** [AriPerkkio/vitest-sonar-reporter#39]
When upgrading to vitest 0.25.5 the tests are failing with: 

```
Error: Failed to load custom Reporter from vitest-sonar-reporter
 ❯ loadCustomReporterModule node_modules/vitest/dist/chunk-vite-node-externalize.ca4784be.js:9194:11
 ❯ async file:/tmp/jhlite/vueapp/node_modules/vitest/dist/chunk-vite-node-externalize.ca4784be.js:9207:32
 ❯ Vitest.setServer node_modules/vitest/dist/chunk-vite-node-externalize.ca4784be.js:10575:170
 ❯ configureServer node_modules/vitest/dist/chunk-vite-node-externalize.ca4784be.js:11395:11
 ❯ createServer node_modules/vite/dist/node/chunks/dep-5605cfa4.js:62117:24
 ❯ createVitest node_modules/vitest/dist/chunk-vite-node-externalize.ca4784be.js:11427:18
 ❯ startVitest node_modules/vitest/dist/chunk-vite-node-externalize.ca4784be.js:11543:15
 ❯ start node_modules/vitest/dist/cli.js:701:17
 ❯ CAC.run node_modules/vitest/dist/cli.js:680:3

Caused by: Error: [vite-node] Failed to load ./sonar-reporter.js
 ❯ ViteNodeRunner.directRequest node_modules/vitest/dist/chunk-vite-node-source-map.d0aa773d.js:237:13
```

Ex of a failing dependabot build: https://github.com/jhipster/jhipster-lite/actions/runs/3646191190/jobs/6157124260

**Comment by AriPerkkio** [comment]
This is fixed in `vitest@0.25.6´, see https://github.com/vitest-dev/vitest/issues/2455.

**Comment by DamnClin** [comment]
Thanks a lot
